package.json

```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "concurrently": "^9.1.0",
     "markdown-toc": "^1.2.0",
-    "playwright": "1.46.0",
+    "playwright": "1.45.0",
```

Comment:
I pulled this down a version because this needs to be in sync with the gem, but sometimes dependabot slips an update through